src/NHibernate.Test/Async/LinqBulkManipulation/Fixture.cs

```diff
@@ -1158,6 +1158,18 @@ public async Task DeleteSyntaxWithCompositeIdAsync()
 			}
 		}
 
+		[Test]
+		public async Task DeleteOnFilterThrowsAsync()
+		{
+			using (var s = OpenSession())
+			using (s.BeginTransaction())
+			{
+				var a = await (s.Query<SimpleEntityWithAssociation>().Take(1).SingleOrDefaultAsync());
+				var query = a.AssociatedEntities.AsQueryable();
+				Assert.That(() => query.Delete(), Throws.InstanceOf<NotSupportedException>());
```

Comment:
@maca88 I've just noticed that async versions of DML extension methods are not used at all in async tests. Is it a bug?

Reply:
Yes it's a bug and I've created [an issue](https://github.com/maca88/AsyncGenerator/issues/121) for it.

Reply:
Published a new release `0.13.2` that has the bug fixed.

Reply:
I do not think we should upgrade the async generator on 5.2x (and even less on previous branches). So we will have to take care of regenerating async files as part of the merge to master I think, in order to take into account the fix now available in master.

Reply:
Same thoughts